peps/pep-0783.rst

```diff
@@ -2,12 +2,13 @@ PEP: 783
 Title: Emscripten Packaging
 Author: Hood Chatham <roberthoodchatham at gmail.com>
 Sponsor: Łukasz Langa <lukasz at python.org>
-Discussions-To:
+Discussions-To: https://discuss.python.org/t/86862
```

Comment:
In future, please only open such threads when the PR has been merged / is about to be merged -- the RtD preview links are temporary, and review can lead to changes in the PEP, meaning the wider audience has an outdated understanding of the text.

We try to put this across in the PR template, but we could strengthen the wording!

A

Reply:
Ah sorry. I opened the thread primarily because I wanted the ci to be green. I figured I could edit the link in the post to point to the merged version when the pr lands.

Reply:
The "Check PEPs" validation error I got reads to me like a direct instruction to go create a discussion thread, so I didn't read anything else before making it:
```
Discussions-To must be a valid thread URL or mailing list
```